src/org/opendatakit/briefcase/ui/PullTransferPanel.java

```diff
@@ -1,536 +1,553 @@
-        }
-      }
-    }
-  }
-
-  private void setTxtOriginEnabled(boolean active) {
-    String strSelection = (String) listOriginDataSource.getSelectedItem();
-    EndPointType selection = (strSelection != null) ? EndPointType.fromString(strSelection)
-        : null;
-
-    if (selection != null) {
-      if (EndPointType.AGGREGATE_0_9_X_CHOICE.equals(selection)
-          || EndPointType.AGGREGATE_1_0_CHOICE.equals(selection)) {
-        txtOriginName.setEditable(false);
-      } else if (EndPointType.CUSTOM_ODK_COLLECT_DIRECTORY.equals(selection)) {
-        txtOriginName.setEditable(active);
-      } else if (EndPointType.MOUNTED_ODK_COLLECT_DEVICE_CHOICE.equals(selection)) {
-        txtOriginName.setEditable(active);
-      } else {
-        throw new IllegalStateException("unexpected case");
-      }
-    }
-  }
-
-  private void setActiveTransferState(boolean active) {
-    setTabEnabled(!active);
-    setTxtOriginEnabled(!active);
-    if (active) {
-      // don't allow normal actions when we are transferring...
-      listOriginDataSource.setEnabled(false);
-      btnOriginAction.setEnabled(false);
-      btnSelectOrClearAllForms.setEnabled(false);
-      btnTransfer.setEnabled(false);
-      // enable cancel button
-      btnCancel.setEnabled(true);
-      // show downloading progress text
-      lblDownloading.setText(DOWNLOADING_DOT_ETC);
-      lblDownloading.setForeground(lblOrigin.getForeground());
-      // reset the termination future so we can cancel activity
-      terminationFuture.reset();
-    } else {
-      // restore normal actions when we aren't transferring...
-      listOriginDataSource.setEnabled(true);
-      btnOriginAction.setEnabled(true);
-      btnSelectOrClearAllForms.setEnabled(true);
-      btnTransfer.setEnabled(true);
-      // disable cancel button
-      btnCancel.setEnabled(false);
-      // hide downloading progress text (by setting foreground color to
-      // background)
-      lblDownloading.setText(DOWNLOADING_DOT_ETC);
-      lblDownloading.setForeground(lblDownloading.getBackground());
-    }
-    // remember state...
-    transferStateActive = active;
-  }
-
-  @EventSubscriber(eventClass = TransferFailedEvent.class)
-  public void failedCompletion(TransferFailedEvent event) {
-    setActiveTransferState(false);
-  }
-
-  @EventSubscriber(eventClass = TransferSucceededEvent.class)
-  public void successfulCompletion(TransferSucceededEvent event) {
-    setActiveTransferState(false);
-  }
-
-  @EventSubscriber(eventClass = FormStatusEvent.class)
-  public void updateDetailedStatus(FormStatusEvent fse) {
-    updateDownloadingLabel();
-  }
-
-  @EventSubscriber(eventClass = RetrieveAvailableFormsFailedEvent.class)
-  public void formsAvailableFromServer(RetrieveAvailableFormsFailedEvent event) {
-    ODKOptionPane.showErrorDialog(PullTransferPanel.this,
-        "Accessing the server failed with error: " + event.getReason(), "Accessing Server Failed");
-  }
-
-}
+/*
```

Comment:
That's exactly what I am thinking, but every character encoding change I tried, made it worse.